Scientist.Test/Scientist.Test.csproj

```diff
@@ -1,22 +1,33 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <TargetFramework>net6.0</TargetFramework>
 
     <IsPackable>false</IsPackable>
+
+    <UserSecretsId>cb167bc0-e4e4-4cb9-97fd-7f8528c2683b</UserSecretsId>
```

Comment:
Not sure about this, only because it has the word secret in it!

Reply:
https://docs.microsoft.com/en-us/aspnet/core/security/app-secrets?view=aspnetcore-5.0&tabs=windows#enable-secret-storage

It is an identifier that can be used when setting secrets for a local run, so I don't think it is an issue to have in the repo

Reply:
Awesome stuff!

Reply:
It defines a ID for your user secrets so we don't have to commit secret information to the repo. Instead the app will look up secrets stored on your machine using this secrets ID.